src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java

```diff
@@ -188,6 +188,7 @@ public void testWriteTagRecordsAndCompactCtors() throws Exception {
             getNonCompilablePath("InputWriteTagRecordsAndCompactCtors.java"), expected);
     }
 
+
     @Override
     protected void verify(Checker checker,
                           File[] processedFiles,
```

Comment:
some of these tests here are not passing because it is using an overridden verify method not the normal one in the abstractTestSupport. any guidance on how to solve this will be helpful

Reply:
I am not an expert on this part of the code, but you may find it useful to look back through git blame to see why these tests are different than others, and investigate the possibility of migrating the tests that deviate to the normal verify methods. In fact, this may even be some prework that you could do before solving this issue.

Reply:
@nrmancuso I looked in git blame didn't help much. but I think that the reason this test is different is that the default severity for this check is info so the `getViolationForFile()` is returning an empty array as info sevirty ( I am not sure please confirm that) . so this verify uses the output stream to test with it not the actual violation returned from the file

Reply:
is migrating back to use `verify` instead of `verifyWithInlineConfigParser` will be a valid solution